pywhat/Data/regex.json

```diff
@@ -163,6 +163,18 @@
           "Networking"
        ]
     },
+    {
+       "Name": "EUI-48 Identifier (Ethernet, WiFi, Bluetooth, etc)",
+       "Regex": "([0-9A-Fa-f]{2}[:]){5}[0-9A-Fa-f]{2}|([0-9A-Fa-f]{2}[-]){5}[0-9A-Fa-f]{2}|([0-9A-Fa-f]{4}[\\.]){2}[0-9A-Fa-f]{4}",
+       "plural_name": false,
+       "Description": null,
+       "Rarity": 0.5,
+       "URL": "https://www.shodan.io/host/",

```

Comment:
Does shodan support EUI-48?

Reply:
Good question.
Initially I used shodan because a quick search of shodan revealed MAC addresses in returned results (and that's what the IP definition uses).  However, on further inspection it doesn't appear that an EUI-48 can actually be used to query shodan, so something else would be better.  https://maclookup.app/ is a free EUI-48 search database that returns valuable results and has a REST API. Unless there are any objections or suggestions I will replace shodan with that.